src/components/layouts/FilterBar.tsx

```diff
@@ -107,12 +107,12 @@ export const FilterBar = ({
 
   const assigneeValue = _assigneeValue as IAssigneeCombined
   return (
-    <Box>
-      <Box
-        sx={{
-          border: { xs: 'none', sm: (theme) => `1px solid ${theme.color.borders.borderDisabled}` },
-        }}
-      >
+    <Box
+      sx={{
+        border: (theme) => `1px solid ${theme.color.borders.borderDisabled}`,
+      }}
```

Comment:
This has given border all around but the border should only be on the bottom according to the Figma design.